mlflow/server/js/src/experiment-tracking/components/CompareRunView.tsx

```diff
@@ -703,4 +749,20 @@ const mapStateToProps = (state: any, ownProps: any) => {
   };
 };
 
+/**
+ * Parse a Python dictionary in string format into a JSON object.
+ * @param value The Python dictionary string to parse
+ * @returns The parsed JSON object, or null if parsing fails
+ */
+const parsePythonDictString = (value: string) => {
+  try {
+    const jsonString = value.replace(/'/g, '"');
+    return JSON.parse(jsonString);
+  } catch (e) {
+    console.error('Failed to parse string to JSON:', e);
+    return null;
+  }
+};
+
+// @ts-expect-error TS(2769): No overload matches this call.
```

Comment:
i think this line can be removed
```suggestion
```

Reply:
It seems we get an error when removing that line:

![image](https://github.com/user-attachments/assets/8ff91e8a-5257-44ec-9bdd-379e4e433952)

Fortunately, I think I managed to fix it when [refactoring the component](https://github.com/mlflow/mlflow/pull/13288)!